Angular/src/app/language-add/language-add.component.html

```diff
@@ -1,3 +1,7 @@
+<button (click)="goBack()" class="button"><i class="material-icons">arrow_back</i></button>
+
 <h1> Add A Language </h1>
 <input class ="input-text" [(ngModel)]="languageName" placeholder="Language Name">
-<a routerLink=".." (click)="addLanguage()"> <i class="material-icons">add_circle_outline</i> </a>
\ No newline at end of file
+<a routerLink=".." (click)="addLanguage()"> \
```

Comment:
Is this backslash supposed to be here?

Reply:
No it is not, good catch. Fixed it in the most recent commit